core/runner/src/test/java/com/cognifide/aet/runner/distribution/dispatch/CollectorJobSchedulerTest.java

```diff
@@ -96,15 +96,19 @@ public void add_expectMessageSent() throws Exception {
   }
 
 
-  @Ignore("Ignored for some unexpected test failure on Linux")
-  @Test(timeout = 2000l, expected = IllegalStateException.class)
-  public void add_whenAddedMessageExists_expectIllegalStateException() throws Exception {
+  @Test(expected = IllegalStateException.class)
```

Comment:
As an alternative to the library, you could consider upgrading to JUnit 5 which has some neat OOTB support for exception testing

    @Test
    void exceptionTesting() {
        Throwable exception = assertThrows(IllegalArgumentException.class, () -> {
            tested.someMethod(illegalArgument);
        });
        assertThat(exception.getMessage(), is("Some expected message"));
    }